services/ui-src/src/components/reports/DrawerReportPage.tsx

```diff
@@ -76,6 +80,13 @@ export const DrawerReportPage = ({ route, validateOnRender }: Props) => {
   const isReportingOnStandards =
     route.path === "/naaar/program-level-access-and-network-adequacy-standards";
 
+  const atLeastOneRequiredAnalysisMethodIsUtilized = report?.fieldData[
+    "analysisMethods"
+  ]?.some(
+    (analysisMethod: AnyObject) =>
+      analysisMethod.analysis_applicable?.[0]?.value === "Yes"
+  );
```

Comment:
this does not account for custom analysis methods, so right now if you select No to all and then add a custom method the error persists. 
this may be a product question: are states allowed to say "No" to all default methods and only have custom methods?

Reply:
confirmed with Anna that as of now, the functionality in this PR is correct